app/src/main/java/ru/vssemikoz/newsfeed/adapters/NewsFeedAdapter.java

```diff
@@ -17,8 +20,10 @@
 public class NewsFeedAdapter extends RecyclerView.Adapter<NewsFeedAdapter.NewsViewHolder> {
 
     private List<NewsItem> newsList;
+    private Context context;
 
-    public void setNewsList(List<NewsItem> newsList) {
+    public void setNewsList(Context context, List<NewsItem> newsList) {
```

Comment:
We should try to limit our parameters list as much as we can. Think about every parameter before adding it:
`context` - do we really need to redefine this variable on each list update? Can't we move it somewhere else?